packages/genesys-spark-tokens/snapshot/css/gse-ui-formsAndInputs.css

```diff
@@ -301,14 +308,9 @@
   --gse-ui-contextMenu-button-default: 32px;
   --gse-ui-contextMenu-button-compact: 24px;
   --gse-ui-contextMenu-borderRadius: 4px;
-  --gse-ui-contextMenu-states-default-foregroundColor: #767c8d;
-  --gse-ui-contextMenu-states-hover-foregroundColor: #3742a2;
-  --gse-ui-contextMenu-states-hover-backgroundColor: #dee1fd;
-  --gse-ui-contextMenu-states-active-foregroundColor: #252e78;
-  --gse-ui-contextMenu-states-active-backgroundColor: #ced2fc;
-  --gse-ui-contextMenu-states-disabled-foregroundColor: #767c8d;
```

Comment:
These are used. Why were they removed?

Reply:
It seems these are not used by UX but used by dev @JackieYada

Reply:
We had a previous conversation that we will use all the tokens from the buttons. So we changed all the colors of the states on the design side to match the button's states.